tests/unit_tests/test_platzky.py

```
@@ -18,7 +18,7 @@ class TestPlatzky:
     def mock_db(self) -> MagicMock:
         return MagicMock()
 
-    def test_change_language_with_domain(self, mock_db):
+    def test_change_language_with_domain(self, mock_db: Any):
```

Ruff (0.14.10)

Dynamically typed expressions (typing.Any) are disallowed in `mock_db`

(ANN401)